build.gradle

```diff
@@ -427,3 +429,12 @@ tasks.named("eclipse").configure {
   dependsOn gradle.includedBuild('build-tools').task(':eclipse')
   dependsOn gradle.includedBuild('build-tools-internal').task(':eclipse')
 }
+
+tasks.register("buildReleaseArtifacts").configure {
+  group = 'build'
+  description = 'Builds all artifacts required for release manager'
+
+  dependsOn subprojects.findAll { it.path.startsWith(':distribution:docker') == false && it.path.startsWith(':ml-cpp') == false }
```

Comment:
When I tested this, I found that I still needed to exclude the task `:x-pack:plugin:ml:bundlePlugin`. I think we'll still need the task exclusion in `release-manager` for that?

However I did find that this change makes #77436 redundant, so what do you think about reverting that?

Reply:
I don't see where we currently exclude `:x-pack:plugin:ml:bundlePlugin`, we only exclude building of the ML native code itself as that's done separately. Am I missing something?

Reply:
> I don't see where we currently exclude `:x-pack:plugin:ml:bundlePlugin`, we only exclude building of the ML native code itself as that's done separately. Am I missing something?

No, I'm missing something, we're OK here 😅